src/components/Question.jsx

```diff
@@ -93,6 +93,33 @@ export default class Question extends React.Component {
     this.props.onChange(this.props.index, newState);
   }
 
+  isDebugged(question) {
+    if (FormUtils.isRelevant(question) && !question[Constants.HAS_SUBQUESTION])
```

Comment:
Not sure why `!question[Constants.HAS_SUBQUESTION]`

Reply:
Removed now